Makefile

```diff
@@ -92,6 +92,9 @@ quicktest:
 quicktest-hsm:
 	docker build --progress=plain -f .docker/Dockerfile-hsm --target test-hsm .
 
+authors:  # updates the AUTHORS file
+	curl https://raw.githubusercontent.com/ory/ci/master/authors/authors.sh | env PRODUCT="Ory Hydra" bash
```

Comment:
* Consider removing `https://github.com/ory/hydra/blob/48603bac894bad157501e609c8f694e8ff17da70/Makefile#L183` if it is made obsolete by this.

* The new implementation ignores `.mailmap` (`%aN`, `%aE`).
* The new implementation doesn't check for shallow repositories. This can be a problem when CI makes commits.

Reply:
Grant!! Thanks for this super helpful review! 

- https://github.com/ory/ci/pull/142 adds the `.mailmap` support
- https://github.com/ory/ci/pull/143 adds the check for complete Git history
- this PR now removes the `contributors` target